### PR TITLE
Hadoop additions to desktop and monitor

### PR DESCRIPTION
**Pull request by arjansh**
Extended the chapter on Hadoop and Spark with instructions on how to register CSV datastores on hdfs and on how to run jobs directly on a Hadoop cluster.

### CODE REVIEW

File: src/docbkx/chapter72-hadoop-interface.xml [hunk 1]
```diff
@@ -291,6 +291,81 @@
...
+                Running jobs on a Hadoop Cluster
+                <inlinemediaobject>
+                    <imageobject>
+                        <imagedata fileref="notice_commercial_editions_only.png" format="PNG" />
+                    </imageobject>
+                </inlinemediaobject>
+            </title>
+
+            <para>
+                To be able to execute jobs from DataCleaner desktop on a Hadoop Cluster you have set the HADOOP_CONF_DIR or YARN_CONF_DIR environment variables (see <link linkend="hadoop_spark_setup">Setting up Spark and DataCleaner environment</link>) and you also have to set the SPARK_HOME environment variable which points to your Apache Spark location. DataCleaner works with the Apache Spark 1.6 releases which can be downloaded from the Apache Spark website. Now you can run DataCleaner jobs on your Hadoop cluster by using the <emphasis>Run on Hadoop cluster</emphasis> option:
```
Comment (jhorcicka) [inline src/docbkx/chapter72-hadoop-interface.xml]: Ignore me if I am wrong but I think there is missing "TO" in: ...DataCleaner desktop on a Hadoop Cluster you have TO set the HADOOP_CONF_DIR...

File: src/docbkx/chapter72-hadoop-interface.xml [hunk 2]
```diff
@@ -291,6 +291,81 @@
...
+					</imageobject>
+				</mediaobject>
+            </para>
+        </section>
+
+        <section>
+            <title>Running jobs on a Hadoop Cluster</title>
+
+            <para>
+                To be able to execute jobs from DataCleaner monitor on a Hadoop Cluster you have set the HADOOP_CONF_DIR or YARN_CONF_DIR environment variables (see <link linkend="hadoop_spark_setup">Setting up Spark and DataCleaner environment</link>) and you also have to set the SPARK_HOME environment variable which points to your Apache Spark location. DataCleaner works with the Apache Spark 1.6 releases which can be downloaded from the Apache Spark website. Now you can run DataCleaner jobs on your Hadoop cluster by going to the schedule dialog and checking the <emphasis>Run on Hadoop cluster</emphasis> option:
```
Comment (jhorcicka) [inline src/docbkx/chapter72-hadoop-interface.xml]: Same missing "TO" in: ...Hadoop Cluster you have TO set...

**Comment by arjansh** [comment]
Don't merge yet. I totally overlooked the "Hadoop cluster" tab in the Options dialog which can be used to do some Hadoop related configuration.

**Comment by LosD** [comment]
LGTM!

**Pull request merged**